shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -390,6 +402,14 @@ public boolean performAction(int virtualViewId, int action, Bundle arguments) {
                 mOwner.dispatchSemanticsAction(virtualViewId, Action.PASTE);
                 return true;
             }
+            default:
+                // might be a local context action.
+                for (LocalContextAction value : mLocalContextActions.values()) {
```

Comment:
If `id >= -1`, can we enforce `resourceId = 1232132123 + id` so we can use `mLocalContextActions.get( GetIdFromResourceId( resourceId ) )` here with `int GetIdFromResourceId(resourceID) { return resourceId - 1232132123; }`? (BTW, replace `1232132123` with some `constexpr` :smiley: )